contrib/adaptive-compression/adapt.c

```diff
@@ -42,6 +42,8 @@ static size_t g_streamedSize = 0;
 static unsigned g_useProgressBar = 1;
 static UTIL_freq_t g_ticksPerSecond;
 static unsigned g_forceCompressionLevel = 0;
+static unsigned g_minCLevel = 1;
+static unsigned g_maxCLevel;
```

Comment:
Since `ZSTD_maxCLevel()` is not a compile-time constant, using `g_maxCLevel = ZSTD_maxCLevel()` here results in an error. Instead, I set it to `ZSTD_maxCLevel()` at the beginning of `main()`